src/defaults.js

```diff
@@ -54,7 +54,8 @@ Selectize.defaults = {
 	dropdownParent: null,
 
 	copyClassesToDropdown: true,
-
+  size: false, // must be a number items displayed or height (px, em, rem, vh)
```

Comment:
I think I see what is going here, but it is a little confusing to overload size with multiple options.  It would be better to have two options (or perhaps an object?).  `sizeType`  that would be an enum of `auto` (default), `number of items`, `fixed height`.  and also a `sizeValue` that would take the number to use.

Reply:
Hm i understand, I was inspired by another component that uses the 'size' thermal.

but an object like this seems like a good idea, if i understand correctly, it would look like this:
`
size: {
    sizeType: 'itemsNumber'
    sizeValue: 6
}
`